renderer/components/LoadingBolt/LoadingBolt.js

```diff
@@ -48,43 +51,79 @@ const FullPageGradient = styled(Flex)`
   );
   background-size: 400% 400%;
   animation: ${gradientMotion} 10s ease infinite;
-  pointer-events: none;
 `
 
+const FullHeightContent = styled(Flex)`
+  width: 100%;
+  height: 100%;
+`
+
+const LoadingCloseButtonWrapper = styled(Box)`
+  height: 40px;
+  cursor: pointer;
+  opacity: 0.6;
+  &:hover: {
+    opacity: 1;
+  }
+`
+
+const LoadingCloseButton = ({ onClose }) => (
```

Comment:
We have close button in a couple of places already (for instance in AutopayModalBody). Would be nice to finally extract this and avoid duplicating them.

Reply:
Have extracted this into a new `CloseButton` component.